parlai/messenger/core/message_sender.py

```diff
@@ -315,10 +315,11 @@ def upload_fb_attachment(self, payload):
                     }
                 }
             }
+            f = open(payload['filename'], 'rb')
```

Comment:
Prefer `with open(payload['filename'], 'rb') as f:` syntax, rather than manually opening and closing

Reply:
Yeah totally right, changed to that